src/Components/Programs/styles.js

```diff
@@ -58,6 +57,35 @@ export const Stroke = styled(View,{
     borderBottomWidth:'2px',
     width:'40vw',
     top:'-35px',
-    // zIndex:'10px'
-
-})
\ No newline at end of file
+})
+export const ArrowNavigation = styled(View,{
+    flexDirection:'row',
+    marginTop:'10px',
+    marginBottom:'10px',
```

Comment:
please make it power of 2

Reply:
and please add a link to your deployed gh pages

Reply:
gh-pages link: https://iamkryptonite.github.io/anitab-org.github.io/